src/components/interactive-map.js

```diff
@@ -180,6 +194,16 @@ export default class InteractiveMap extends PureComponent {
     // Register additional event handlers for click and hover
     eventManager.on('mousemove', this._onMouseMove);
     eventManager.on('click', this._onMouseClick);
+
+    // run stub queued action
+    this._eventManager.queue.forEach(({events, ref}) => {
+      if (ref) {
```

Comment:
`ref` is an optional argument. This implementation relies on `ref` being defined when the child components register its events. It would be safer to use another flag to differentiate `on` and `off`?

Reply:
yes, you are right, will fix.